app/controllers/webhooks_controller.rb

```diff
@@ -235,6 +270,27 @@ def generate_seller_gift_card_id_hash
     ('a'..'z').to_a.sample(5).join
   end
 
+  # rubocop:disable Layout/LineLength
+  def send_pool_donation_receipt(payment_intent:, amount:)
+    amount_string = format('%.2f', (amount.to_f / 100))
```

Comment:
mind running `rubocop -a` and resolving?

Reply:
Do you mind if I come back and do it after? It's going to auto-fix a bunch of stuff that's not related to this PR and I don't want the logic to get lost in the mix

Reply:
I fixed all the stuff in the relevant files!

Reply:
sgtm!